edge-util/src/Microsoft.Azure.Devices.Edge.Util/SslProtocolsHelper.cs

```diff
@@ -47,15 +47,6 @@ public static SslProtocols Parse(string protocols, SslProtocols defaultSslProtoc
         public static string Print(this SslProtocols sslProtocols)
         {
             var sslProtocolsList = new List<string>();
-            if ((sslProtocols & SslProtocols.Ssl2) > 0)
-            {
-                sslProtocolsList.Add($"{SslProtocols.Ssl2}");
-            }
-
-            if ((sslProtocols & SslProtocols.Ssl3) > 0)
-            {
-                sslProtocolsList.Add($"{SslProtocols.Ssl3}");
-            }
```

Comment:
reminder: confirm removing these is necessary (if I recall correctly dotnet 3 deprecated something)

Reply:
Both Ssl2 and Ssl3 have been superseded by the TLS protocol, https://docs.microsoft.com/en-us/dotnet/api/system.security.authentication.sslprotocols?view=netcore-3.1